server/src/test/java/io/crate/expression/operator/OrOperatorTest.java

```diff
@@ -22,13 +22,24 @@
 package io.crate.expression.operator;
 
 import static io.crate.testing.Asserts.isLiteral;
+import static io.crate.testing.Asserts.isReference;
 
 import org.junit.Test;
 
 import io.crate.expression.scalar.ScalarTestCase;
 
 public class OrOperatorTest extends ScalarTestCase {
 
+    @Test
+    public void test_normalize_boolean_true_or_non_literal() throws Exception {
```

Comment:
Just a naming suggestion, but your call if you prefer
```suggestion
    public void test_normalize_boolean_true_or_reference() throws Exception {
```

or 

```suggestion
    public void test_normalize_boolean_true_or_column() throws Exception {
```